docs/glossary.md

```diff
@@ -0,0 +1,97 @@
+# Glossary
+
+Definition of commonly used terms in the context of this Romcal project. The terms are listed in alphabetical order.
+
+## Celebration
+
+A celebration is a liturgical or sacramental event, typically consisting of rituals, readings, prayers, and often the Eucharist (Mass), whether or not it is associated with a liturgical day.
+
+Note: Currently, Romcal does not provide specific information about celebrations, such as the list of Masses for a given day or the hours of the Divine Office. Instead, it indicates unique liturgical days determined within a liturgical calendar. However, the ability to list these celebrations—linked to a liturgical day or corresponding to a civil day (such as Vigil Masses)—is a feature that may be added in the future.
+
+## General Instruction of the Liturgy of the Hours (GILH)
```

Comment:
```suggestion
## General Instruction of the Liturgy of the Hours
```

Reply:
Here, I think it's important to keep the acronym in the title to keep it prominent. Because this acronym is what we'll use elsewhere to refer to this document.

Reply:
agreed

Reply:
Well, I consider a glossary item to be similar to a dictionary entry from the formal/structure point of view: the headword in a dictionary entry should not include anything else but the word defined only.

Same here: while we don’t place definition of a glossary item on the same line as the term name, the term name (heading in our case) should not include anything but the term name.

For abbreviations/acronyms/initialisms we should add an additional glossary item where we point the reader to the proper place using `See [Some Term](#some-term)`. Etienne, you already do this for some of the items, e.g. [Sanctoral](https://github.com/romcal/romcal/blob/e17e05a96a63e98771e8a6c0d114f92cb59b0297/docs/glossary.md#sanctoral) refers to the [Proper of Saints](https://github.com/romcal/romcal/blob/e17e05a96a63e98771e8a6c0d114f92cb59b0297/docs/glossary.md#proper-of-saints).

Therefore, I don’t think it is a good idea to append abbreviations/acronyms/initialisms to main glossary item names. However, we should add it after the first use of the term in its definition (again, like you’ve already done it in [Proper of Saints](https://github.com/romcal/romcal/blob/e17e05a96a63e98771e8a6c0d114f92cb59b0297/docs/glossary.md#proper-of-saints)).

Reply:
i'll go through (in a few hours) and add an upper section of abbreviations to cover our concerns about abbreviations being lost, that way they don't have to be in these titles and can be linked as the abbreviations in the titles with a `see -> #longer-title-form` in the glossary.